packages/ibm-products-web-components/src/components/interstitial-screen/interstitial-screen.stories.ts

```diff
@@ -0,0 +1,503 @@
+      >
+        <c4p-interstitial-screen-header
+          header-title="Use case-specific title"
+          header-subtitle="Use case-specific sub title"
+        ></c4p-interstitial-screen-header>
+        <c4p-interstitial-screen-body>
+          ${getMultipleContent()}
+        </c4p-interstitial-screen-body>
+        <c4p-interstitial-screen-footer
+          ?async-action=${true}
+          @c4p-on-action="${onAction}"
+        ></c4p-interstitial-screen-footer>
+      </c4p-interstitial-screen>
+    `;
+  },
+};
+
+export const WithCustomActionButtons = {
+  render: (args) => {
+    const blockClass = `${prefix}--interstitial-screen`;
+    const container = document.createElement('div');
+    let config;
+    const stepLength = 4;
+    const progStepCeil = stepLength - 1;
+    let initialized = false;
+    let currentStep = 0;
+
+    const handleAction = async (actionType) => {
+      const carouselAPI = config?.carouselAPI;
+
+      if (!carouselAPI) {
+        return;
+      }
+
+      if (actionType === 'next') {
+        carouselAPI.next();
+        currentStep = currentStep + 1;
+      } else if (actionType === 'back') {
+        carouselAPI.prev();
+        currentStep = currentStep - 1;
+      } else {
+        // handle 'skip' or 'submit' action here
+        document
+          .querySelector('c4p-interstitial-screen')
+          ?.toggleAttribute('open');
+      }
+
+      rerender();
+    };
+
+    const handleSkip = () => handleAction('skip');
+    const handleStart = () => handleAction('start');
+    const handleClickNext = () => handleAction('next');
+    const handleClickPrev = () => handleAction('back');
+
+    const rerender = () => {
+      litRender(
+        html`
+          <div id="page-content-selector">
+            <cds-button @click="${toggleButton}"
+              >Show Interstitial modal</cds-button
+            >
+          </div>
+          <c4p-interstitial-screen
+            ?fullscreen=${false}
+            ?open=${true}
+            @c4p-interstitial-opened=${(e) => {
+              if (!initialized) {
+                config = e.detail;
+                initialized = true;
+                rerender();
+              }
+            }}
+          >
+            <c4p-interstitial-screen-header
+              header-title="Use case-specific title"
+              header-subtitle="Use case-specific sub title"
+            ></c4p-interstitial-screen-header>
+
+            <c4p-interstitial-screen-body>
+              ${getMultipleContent()}
+            </c4p-interstitial-screen-body>
+
+            <c4p-interstitial-screen-footer>
+              <style>
+                ${footerStyles} cds-button {
+                  block-size: 100%;
+                  display: contents;
+                }
+              </style>
+
+              <div class="${blockClass}--footer">
+                <cds-button
+                  class="${blockClass}--skip-btn"
+                  kind="ghost"
+                  size="lg"
```

Comment:
I think these should all use the larger button size? It also might help to differentiate the text here like the React version does because at first I didn’t realize what made these buttons custom other than the size, which felt more like a bug.

```suggestion
                  size="xl"
```